core/frontend/src/ViewState.ts

```diff
@@ -256,6 +257,15 @@ export abstract class ViewState extends ElementState {
    */
   public static createFromProps(_props: ViewStateProps, _iModel: IModelConnection): ViewState | undefined { return undefined; }
 
+  /** Serialize this ViewState as a set of properties that can be used to recreate it via [[ViewState.createFromProps]]. */
+  public toProps(): ViewStateProps {

```

Comment:
Don't we usually call this toJSON?

Reply:
`toProps` is the inverse of `createFromProps` as `toJSON` is the inverse of `fromJSON`.